mrp_workcenter_cost/tests/test_mrp_workcenter_cost.py

```diff
@@ -0,0 +1,75 @@
+# Copyright 2023 Camptocamp SA (https://www.camptocamp.com).
+# @author Iván Todorovich <ivan.todorovich@camptocamp.com>
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo.tests import Form, TransactionCase
+
+
+class TestMrp(TransactionCase):
+    @classmethod
+    def setUpClass(cls):
+        super().setUpClass()
+        cls.env = cls.env(context=dict(cls.env.context, tracking_disable=True))
+        cls.bom = cls.env.ref("mrp.mrp_bom_table_top")
```

Comment:
```suggestion
        cls.bom = cls.env.ref("mrp.mrp_bom_table_top")
        # Activate operations disabled by default in demo data
        cls.bom.with_context(active_test=False).operation_ids.active = True
```
Cf https://github.com/odoo/odoo/commit/aaba4eb4deb3f9e10fd611924c5a24680d633537

Reply:
Thank you! It's working now :+1: